<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DockWelcome</class>
 <widget class="QWidget" name="DockWelcome">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>16</number>
     </property>
     <item>
      <widget class="QLabel" name="lIcon">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string notr="true">&lt;Icon&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lWelcome">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>64</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>32</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Welcome to ScheduleMaster</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Policy::Maximum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pbToggleNews">
         <property name="text">
          <string>Hide News</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
     <item>
      <widget class="QGroupBox" name="gbActions">
       <property name="title">
        <string>Actions</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCommandLinkButton" name="clbNewProject">
          <property name="text">
           <string>New Project</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources.qrc">
            <normaloff>:/Icons/file-blank.ico</normaloff>:/Icons/file-blank.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="clbOpenProject">
          <property name="text">
           <string>Open Project from File</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources.qrc">
            <normaloff>:/Icons/folder-open.ico</normaloff>:/Icons/folder-open.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>232</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="clbPlugins">
          <property name="text">
           <string>Plugins</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources.qrc">
            <normaloff>:/Icons/puzzle-piece.ico</normaloff>:/Icons/puzzle-piece.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="clbPreferences">
          <property name="text">
           <string>Preferences</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources.qrc">
            <normaloff>:/Icons/gear.ico</normaloff>:/Icons/gear.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="clbHelp">
          <property name="text">
           <string>Help</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources.qrc">
            <normaloff>:/Icons/book-manual.ico</normaloff>:/Icons/book-manual.ico</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCommandLinkButton" name="clbQuit">
          <property name="text">
           <string>Quit</string>
          </property>
          <property name="icon">
           <iconset resource="../Resources.qrc">
            <normaloff>:/Icons/circle-xmark.ico</normaloff>:/Icons/circle-xmark.ico</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbRecentFiles">
       <property name="title">
        <string>Recent Projects</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QListWidget" name="lwRecentProjects">
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbOpen">
            <property name="text">
             <string>Open</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="gbNews">
       <property name="title">
        <string>News</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListWidget" name="lwNews">
          <property name="minimumSize">
           <size>
            <width>450</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>450</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbMoreNews">
            <property name="text">
             <string>Read more News...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
